Task/Taskfile.yml

```diff
@@ -241,6 +246,7 @@ tasks:
       - find {{.ROOT_DIR}} -type d -name '.task' -exec rm -rf {} +
       - find {{.ROOT_DIR}} -type f -name 'sbom.*.json' -delete
       - find {{.ROOT_DIR}} -type f -name 'vulns.*.json' -delete
+      - find {{.ROOT_DIR}} -type f -name 'seiso_*:*.tar' -delete
```

Comment:
Add the OCI artifact to the cleanup. A little too much `*` but I didn't want to require downstream projects pass in the project name via their `Taskfile.yml` in `vars` just to refine this cleanup to have i.e. `seiso_goat` for the goat project. If it's an issue in the future we can add that.